tests/integrations/test_dbm.py

```diff
@@ -5,7 +5,7 @@
 from utils import weblog, interfaces, context, missing_feature, released, scenarios
 
 
-@released(cpp="?", golang="?", java="?", nodejs="?", dotnet="2.26.0", php="?", ruby="?")
+@released(cpp="?", golang="?", java="?", nodejs="?", php="?", ruby="?")
```

Comment:
Hi I'm new here and don't know what the `@released` annotation does. Was it intentional to remove the `dotnet` entry?

Reply:
Sounds good, thanks for the explanation